tests/integration_tests/conftest.py

```diff
@@ -118,17 +118,30 @@ def setup_image(session_cloud: IntegrationCloud, request):
     """
     source = get_validated_source(session_cloud)
     if not (
-        source.installs_new_version() or integration_settings.INCLUDE_COVERAGE
+        source.installs_new_version()
+        or integration_settings.INCLUDE_COVERAGE
+        or integration_settings.INCLUDE_PROFILE
     ):
         return
     log.info("Setting up source image")
     client = session_cloud.launch()
     if source.installs_new_version():
         log.info("Installing cloud-init from %s", source.name)
         client.install_new_cloud_init(source)
+    if (
+        integration_settings.INCLUDE_PROFILE
+        and integration_settings.INCLUDE_COVERAGE
+    ):
+        log.error(
+            "Invalid configuration, cannot enable both profile and coverage."
+        )
+        raise ValueError()
```

Comment:
Nit: Could we also include the error message in the exception? Just in case, the logs are not visible to the user.

```suggestion
        raise ValueError("Invalid configuration, cannot enable both profile and coverage.")
```